setup/module/api/user_data.sh

```diff
@@ -1,9 +1,98 @@
+Content-Type: multipart/mixed; boundary="==BOUNDARY=="
+MIME-Version: 1.0
+
+--==BOUNDARY==
 #!/bin/bash
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
 echo ECS_CLUSTER=${cluster_id} >> /etc/ecs/ecs.config
 echo ECS_ENGINE_AUTH_TYPE=dockercfg >> /etc/ecs/ecs.config
-yum install -y aws-cli awslogs
+echo ECS_LOGLEVEL=debug >> /etc/ecs/ecs.config
```

Comment:
🤦‍♂️  windows...

Reply:
This is for the L0 API module specifically which is always going to be a Linux ECS container. I think I want to do something separate for Windows and Linux environments that are created.

Reply:
🤦‍♂️ linux...